src/main/java/org/sagebionetworks/web/client/Portal.java

```diff
@@ -72,6 +73,28 @@ public void onSuccess() {
               // make sure jsni utils code is available to the client
               ginjector.getSynapseJSNIUtils();
 
+              if (DisplayUtils.isInTestWebsite(ginjector.getCookieProvider())) {
+                ginjector
+                  .getSynapseJavascriptClient()
+                  .getFeatureFlagConfig(
+                    new AsyncCallback<String>() {
+                      @Override
+                      public void onSuccess(String config) {
+                        ginjector
+                          .getCookieProvider()
+                          .setCookie("PORTAL_FEATURE_FLAG", config);
+                      }
```

Comment:
Only store the configurations inside cookies if portals is under experimental mode